hydra/_internal/core_plugins/bash_completion.py

```diff
@@ -33,7 +33,16 @@ def install(self) -> None:
         true
     fi
 
-    if ! [ -x "$(command -v $helper)" ]; then
+    EXECUTABLE=($(command -v $helper))
+    if [ "$HYDRA_COMP_DEBUG" == "1" ]; then
+        printf "\\n"
+        printf "helper='$helper'\\n"
+        printf "EXECUTABLE='$EXECUTABLE'\\n"
+        printf "EXECUTABLE_FIRST='${EXECUTABLE[0]}'\\n"
+        printf "EXECUTABLE AS ARRAY:\\n"
+        printf "\\t%s\\n" ${EXECUTABLE[@]}
```

Comment:
make those print just one line.

Reply:
Just merged 2 lines into 1 - is that fine?
Merging others would break clarity of the code (to my view).